articles/architecture-scenarios/b2c/authentication.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Authentication
+description: Understand how authentication works in your CIAM implementation. 
+toc: true
+topics:
+    - b2c
+    - ciam
+    - authentication
+    - universal-login
+contentType: concept
+useCase:
+    - authentication
+---
+# User Authentication
+
+In order to provide services to your users, you must be able to identify who those users are. This process is called authentication. You must provide your users a way to verify that they are who they say they are. There are a number of ways to do this (such as social media accounts, username/password, passwordless authentication). In most cases it is also recommended that you go beyond a first factor for authenticating the user and add a second factor as well.
+
+When designing my authentcation experience, how do I...?
+
+* ...determine where users will enter their credentials?
+* ...keep user credentials safe?
+* ...minimize the upkeep on my authentication system?
+* ...provide password authentication for my users?
+* ...integrate social logins for my users?
+* ...prevent hackers from trying to log in as my users?
+* ...implement authentication in different kinds of applications using Auth0?
+* ...make login easy for my users? I have customers from different language backgrounds?
+* ...provide a good user experience as I migrate away from my legacy authentication system?
+
+The answers to these questions, and others, are detailed below in sections on [Universal Login](#universal-login), [Username Password Authentication](#username-password-authentication), [Anomaly Detection](#anomaly-detection), and [Application Integration](#application-integration).
```

Comment:
I think rather than add even more non-content than is already here (topic indices basically) I'd rather cut that paragraph to "The answers to these questions and more are detailed below". WDYT?

Reply:
The intention behind this document is to explicitly guide the reader. The problem with "the answers to these questions and more are detailed below" is that the reader has to hunt through all the sections until they find the answer to the question. The verbiage in the original B2C Welcome was - rather than being specific content - actually designed to be narrative: the reader only has to go through one (or two) paragraphs until they find where to go to get the question answered. Does that make sense?

Reply:
Rewrite coming on all these to standardize the tone. I will make note of navigation to the pertinent paragraphs.